scapy/layers/inet6.py

```diff
@@ -3044,6 +3044,12 @@ class MIP6MH_Generic(_MobilityHeader):  # Mainly for decoding of unknown msg
                    StrLenField("msg", b"\x00" * 2,
                                length_from=lambda pkt: 8 * pkt.len - 6)]
 
+    @classmethod
+    def dispatch_hook(cls, _pkt=None, *_, **kargs):
```

Comment:
Is this related to this PR?

Reply:
I tried at first to properly dissect the HC1 IPv6 extensions but the padding is not bit round (some layers are 20 bits with no padding), so it's not really supported.
This was used to do that. Despite not being used in the end, I thought it still made sense

Reply:
If this is not used, I think that it is better to remove this function.